overlay.py

```diff
@@ -159,25 +41,15 @@ def __init__(self, bike, width=1280, height=740):
 		self.set_callback_for_topic_list(self.data.get_topics(), self.on_data_message)
 		self.set_callback_for_topic_list([str(DAShboard.recording)], self.on_recording_message)
 
-	def show_opencv_frame(self):
-		""" Creates the frame using the webcam and canvases, and displays result """
-		_, frame = self.webcam.read()
-		frame = cv2.resize(frame, (self.width, self.height))
-
-		frame = self.base_canvas.copy_to(frame)
-		frame = self.data_canvas.copy_to(frame)
-		frame = self.message_canvas.copy_to(frame)
-
-		cv2.imshow('frame', frame)
-		cv2.waitKey(self.frametime)
+	def publish_recording_status(self, message: str) -> None:
```

Comment:
`Backend` needs to somehow be able to trigger an MQTT message with the recording status. Passing the MQTT client instance down into the class felt wrong, does this solution seem good? It doesn't feel 100% ideal to me but perhaps it's fine

Reply:
I like the idea of this. I need to have a think about it, and whether it belongs in this PR

Reply:
Nah, complete it in another PR I reckon.